docs/deployment/feature_gates.md

```diff
@@ -94,6 +92,9 @@ The following tables are a summary of the feature gates that you can set on diff
 | WorkerPoolKubernetesVersion                  | `true`  | `Beta`    | `1.46` | `1.49` |
 | WorkerPoolKubernetesVersion                  | `true`  | `GA`      | `1.50` | `1.51` |
 | WorkerPoolKubernetesVersion                  | `true`  | `Removed` | `1.52` |        |
+| DisableDNSProviderManagement                 | `false` | `Alpha`   | `1.41` | `1.49` |
+| DisableDNSProviderManagement                 | `true`  | `Beta`    | `1.50` | `1.51` |
+| DisableDNSProviderManagement                 | `true`  | `GA`      | `1.52` |        |
```

Comment:
@acumino Are you willing to add this PR to v1.52? If not, @MartinWeindel needs to adapt the docs here.

Reply:
It's better if we add this PR in v1.52. If required changes are done I will add it to v1.52 milestone.